rdmo/management/imports.py

```diff
@@ -109,25 +124,27 @@ def import_element(
     perms_error_msg = check_permissions(instance, uri, user)
     if perms_error_msg:
         # when there is an error msg, the import can be stopped and return
-        element["errors"].append(perms_error_msg)
+        element[ImportElementFields.ERRORS].append(perms_error_msg)
         return element
 
-    element['created'] = created
-    element['updated'] = not created and original is not None
+    element[ImportElementFields.CREATED] = created
+    element[ImportElementFields.UPDATED] = not created and original is not None
     # INFO: the dict element[FieldNames.diff.value] is filled by calling track_changes_on_element
 
     element = strip_uri_prefix_endswith_slash(element)
     # start to set values on the instance
-    apply_field_values(instance, element, import_helper, uploaded_uris, original)
+    apply_field_values(instance, element, import_helper, original)
 
     # call the validators on the instance
     validate_instance(instance, element, *import_helper.validators)
 
-    if element.get('errors'):
+    update_extra_fields_from_validated_instance(instance, element, import_helper, original=original)
+
+    if element.get(ImportElementFields.ERRORS):
         # when there is an error msg, the import can be stopped and return
         if save:
-            element['created'] = False
-            element['updated'] = False
+            element[ImportElementFields.CREATED] = False
+            element[ImportElementFields.UPDATED] = False
```

Comment:
I think the `ImportElementFields` class was added before, but its is not nessesary, and add another layer of complexity.

Reply:
yeah, Ive started using that `StrEnum` `ImportElementFields` and added here for consistency I think.
I don't think it's more complexity because it uses constants that can be easily tracked instead of string literals..